grafana/resource_stack.go

```diff
@@ -0,0 +1,271 @@
+package grafana
+
+import (
+	"context"
+	"log"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+
+	gapi "github.com/grafana/grafana-api-golang-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+var stackSlugRegex = regexp.MustCompile("^[a-z][a-z0-9]+$")
+
+func ResourceStack() *schema.Resource {
+	return &schema.Resource{
+
+		Description: `
+* [Official documentation](https://grafana.com/docs/grafana-cloud/reference/cloud-api/#stacks/)
+* [HTTP API](https://grafana.com/docs/grafana/latest/http_api/stack/)
```

Comment:
This link returns 404 for me.

Reply:
@dohnto Thank you! I put that as a placeholder. What would be the right link? (I couldn't find API docs for stacks apart from the first link).

Reply:
@julienduchesne Shall I remove the `HTTP API` link given that are no docs for it currently?

Reply:
Yes